docker-utils/startup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+
+cfgTpl=/usr/local/bin/custom-cfg.sh
+cfgHome=/usr/local/zeebe/conf/zeebe.cfg.toml
+cfgLog=/usr/local/zeebe/conf/log4j2.xml
+
+if [[ "$DEPLOY_ON_KUBERNETES" == "true" ]]; then
+
+    DNSNAME="${HOST}.${DNS}"
+
+    sed -i "s/@DNSNAME@/${DNSNAME}/g" $cfgHome
+
+    if [[ -n $INITIAL_CONTACT_POINT && ! $HOST =~ .*-0$ ]]; then
+        sed -i "s/@INITIAL_CONTACT_POINT@/initialContactPoints = \[\n\t\"${INITIAL_CONTACT_POINT}\"\n\]/g" $cfgHome
+    else
+        sed -i "s/@INITIAL_CONTACT_POINT@//g" $cfgHome
+    fi
+
+else
+
+    sed -i "s/@DNSNAME@/0.0.0.0/g" $cfgHome
```

Comment:
Can we use the local IP address instead of `0.0.0.0` here, this would allow users to connect from outside of the container. Maybe we could use `hostname --ip-address` to get the local address?

Reply:
I tried with `0.0.0.0` and `hostname --ip-address` and I was able in both cases to zeebe and create a test task. Should i replace it anyway?

Reply:
`0.0.0.0` will only work if you run the client outside of the docker container on localhost with port mapping of `51015:51015`. The `hostname --ip-address` should also work if you don't have an equal port mapping and run the client in another docker container, or have  a cluster of nodes on localhost. So I would prefer the `hostname --ip-address` approach.